streamwebs_frontend/streamwebs_frontend/settings.py.dist

```diff
@@ -169,29 +168,6 @@ SCHOOL_CHOICES = (
     ('a', 'School A'),
 )
 
-# Pipeline settings for static files
-# Used to package, compress, and minimize our bower dependencies (e.g. JQuery)
-# https://axiacore.com/blog/effective-dependency-management-django-using-bower/
-# http://django-pipeline.readthedocs.io/en/latest/installation.html
-
-STATICFILES_STORAGE = 'pipeline.storage.PipelineCachedStorage'
-STATICFILES_FINDERS = (
```

Comment:
Isn't this necessary for normal static file mgmt? I think just the pipeline line should be removed.